src/main/java/de/t0biii/music/commands/CMD_PlayMusic.java

```diff
@@ -25,47 +25,85 @@ public boolean onCommand(CommandSender sender, Command cmd, String commandLabel,
     if (sender.hasPermission("JoinMusic.use")) {
       if (sender instanceof Player) {
         Player player = ((Player) sender).getPlayer();
-        String noperm = plugin.getConfig().getString("messages.no-permission");
+        String noperm = plugin.getConfig().getString("messages.no-permission").replaceAll("&", "§");
         if (args.length == 1) {
           if (args[0].equalsIgnoreCase("reload")) {
             if (sender.hasPermission("JoinMusic.command.reload")) {
               try {
-                plugin.reloadConfig();
+                plugin.reloadConfigs();
                 player
                     .sendMessage(plugin.prefix + ChatColor.DARK_AQUA + plugin.getConfig().getString("messages.reload"));
               } catch (Exception exception) {
                 player.sendMessage(plugin.prefix + ChatColor.RED + "Reload failed!");
               }
             } else {
-              sender.sendMessage(plugin.prefix + ChatColor.RED + noperm);
+              sender.sendMessage(plugin.prefix + noperm);
             }
           } else if (args[0].equalsIgnoreCase("stop")) {
             if (sender.hasPermission("JoinMusic.command.stop")) {
               Music.stop(player);
-              player.sendMessage(plugin.prefix + ChatColor.DARK_AQUA + plugin.getConfig().getString("messages.stop"));
+              player.sendMessage(plugin.prefix + plugin.getConfig().getString("messages.stop").replaceAll("&", "§"));
             } else {
-              sender.sendMessage(plugin.prefix + ChatColor.RED + noperm);
+              sender.sendMessage(plugin.prefix + noperm);
             }
-          } else {
+          } else if (args[0].equalsIgnoreCase("disable") && plugin.getConfig().getBoolean("options.allowDisabling")) {
+        	  if(sender.hasPermission("JoinMusic.command.disableOwn")) {
+        		plugin.getUserConfigs().set(player.getUniqueId().toString(),true);
+        		sender.sendMessage(plugin.prefix + plugin.getConfig().getString("messages.disabled").replaceAll("&", "§"));
+       		    plugin.saveUserConfigs();
+        	  }else {
+        		sender.sendMessage(plugin.prefix + noperm);
+        	  }
+          }else if (args[0].equalsIgnoreCase("enable") && plugin.getConfig().getBoolean("options.allowDisabling")) {
+        	  if(sender.hasPermission("JoinMusic.command.disableOwn")) {
+        		// Preventing to set it to enabled if it doesn't exist, saving file size
+        		// It's not possible to just remove it with FileConfiguration I think
+        		if(plugin.getUserConfigs().isSet(player.getUniqueId().toString())) {
+        		  plugin.getUserConfigs().set(player.getUniqueId().toString(),false);
+        		  plugin.saveUserConfigs();
```

Comment:
Setting it to null removes the entry from the file:
https://www.spigotmc.org/threads/delete-configuration-section.131046/